lesson5/e-commerce-2/main.ts

```diff
@@ -29,12 +29,11 @@ aj.joinWebsite(to)
 aj.joinWebsite(bim)
 aj.joinWebsite(nu)
 
-//aj.signInSession(nu)
-//aj.signOutSession(bim)
-
-nightMarket.findProductByName(nu, "Iphone 15 Pro")
-//dayMarket.searchProducts(nu)
-
 
+aj.addProduct(macbook, aj.signInSession(to));
```

Comment:
User can only interact with website. Not with inner components (markets, products, employees,...)